dashboard/components/BannerCampaign.vue

```diff
@@ -17,6 +17,15 @@
 				>
 					{{ campaign.campaign }}
 				</a>
+				<a
+					class="banner-actions-icon"
```

Comment:
A suggestion for further improving this: With pull request overview URL `https://github.com/wmde/fundraising-banners/pulls` being the default, you could make `class`, `title` and `data-tooltip` dynamic to indicate if there is a "real" PR or just the overview and adjust the color/tooltip-text of the link. Or remove the link with a `v-if="pullRequestUrl==='https://github.com/wmde/fundraising-banners/pulls'"`. I';; leave the decision up to you.

Reply:
Done.